maxmckenzie/mica
# handling folder clash on install

# Status
ready

# Description
https://github.com/maxmckenzie/mica/issues/20
Feature cleverer way to deal with folder name conflict


# Check List
- [] `npm run lint` passes

# Deployment
- [] `npm run build` to build the /dist dir ready for push to npm

## Deploy Notes
npm publish

## Steps to Test or Reproduce
```
Given I am installing a test framework with mica install
And I already have a ./test folder 
Then I should be prompted for input about how to proceed
```
Expected behaviour:

Currently you see the error message  fatal: destination path './test' already exists and is not an empty directory. and installation is halted.

```sh
git pull
git checkout <feature_branch>
```
